mne/externals/pymatreader/utils.py

```diff
@@ -51,6 +51,15 @@ def _import_h5py():
     return h5py
 
 
+def get_matfile_version():
+    """Triage import path based on SciPy version."""
+    try:
+        from scipy.io.matlab.miobase import get_matfile_version
+    except ModuleNotFoundError:  # scipy 1.8
+        from scipy.io._matlab.miobase import get_matfile_version
+    return get_matfile_version
```

Comment:
This might actually be a good function to have in the `scipy.io.matlab` namespace, any interest in making a PR to SciPy to add it @drammock ?

Reply:
I already made what I think are the needed code changes, but I don't have the time right now to get a build/test env working for scipy.  So: nope.

Reply:
https://github.com/scipy/scipy/pull/15004